documentatie/gebruikersdocumentatie/instructies-invoerder.md

```diff
@@ -2,20 +2,34 @@
 
 Welkom bij Abacus! Deze tool is ontwikkeld om de verkiezingsuitslagen per stembureau na te rekenen en te controleren op fouten en afwijkende uitslagen. In dit document leggen we uit hoe je Abacus gebruikt als invoerder.
 
-Na het tellen van de stemmen wordt voor elk stembureau een proces-verbaal ingevuld. De cijfers op dit proces-verbaal neem je over in Abacus. Vervolgens controleert de applicatie of de cijfers kloppen, en als alles klopt kun je de resultaten opslaan. Als er fouten of waarschuwingen zijn, volg je de instructies op of overleg je het met je coördinator.
+Na het tellen van de stemmen wordt voor elk stembureau een proces-verbaal ingevuld. De cijfers op dit proces-verbaal neem je over in Abacus. Vervolgens controleert de applicatie of de cijfers kloppen met het proces-verbaal. Ook wordt gecheckt of er bijzondere afwijkingen zijn, bijvoorbeeld als het aantal ongeldige stemmen hoog is. Als er fouten of waarschuwingen zijn, volg je de instructies op of overleg je het met je coördinator.
```

Comment:
```suggestion
Na het tellen van de stemmen wordt voor elk stembureau een proces-verbaal ingevuld. De cijfers op dit proces-verbaal neem je over in Abacus. Vervolgens controleert de applicatie of de cijfers intern kloppen, bijvoorbeeld of de optellingen goed zijn. Ook wordt gecheckt of er bijzondere afwijkingen zijn, bijvoorbeeld als het aantal ongeldige stemmen hoog is. Als er fouten of waarschuwingen zijn, volg je de instructies op of overleg je het met je coördinator.
```
Abacus niet controleren of de cijfers kloppen met papier, want Abacus kan het papier niet lezen :-) We doen alleen een check op interne consistentie. Voorstel om dat begrijpelijk uit te leggen in de suggestie.